subsys/bluetooth/shell/gatt.c

```diff
@@ -215,6 +214,7 @@ static int cmd_discover(const struct shell *sh, size_t argc, char *argv[])
 {
 	int err;
 
+	struct bt_uuid_16 uuid;
```

Comment:
have you tested it? I think it's not as clear-cut. it needs to stay global/static as it's used outside of those functions. Just renaming it would be fine IMO.

Reply:
Only these two places directly use global variables, which are only used to receive parameters from the command line and to call function parameters

Reply:
yep you're right, I'd still be more comfortable if you tried those two commands in the shell just in case.

Reply:
OK, I'll do a test later. In order to avoid blocking, rename it first